nestkernel/connection_manager.cpp

```diff
@@ -134,7 +134,7 @@ void
 nest::ConnectionManager::set_status( const DictionaryDatum& d )
 {
   long init_cap = init_conn_capacity_;
-  if ( updateValue< long >( d, "init_connector_capacity", init_cap ) )
+  if ( updateValue< long >( d, names::init_connector_capacity, init_cap ) )
```

Comment:
I think I would change this name to `initial_connector_capacity`.  Just `init_...` evokes too strong associations to the verb "initialize" and may therefore be confusing.